instrumentation/sinatra/Appraisals

```diff
@@ -4,8 +4,12 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-appraise 'sinatra-3.0.x' do
-  gem 'sinatra', '~> 3.0.0'
+appraise 'sinatra-4.x' do
+  gem 'sinatra', '~> 4.0'
+end
+
+appraise 'sinatra-3.x' do
+  gem 'sinatra', '~> 3.0'
```

Comment:
was there any reason this was stricter before?

Reply:
No real reason. We don't have a consistent pattern or way to maintain the appraisals.